.github/workflows/CI.yaml

```diff
@@ -0,0 +1,89 @@
+name: Continuous Integration
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'docker/**'
+      - 'graphenex/**'
+  push:
+    branches:
+      - main
+    paths:
+      - 'docker/**'
+      - 'graphenex/**'
```

Comment:
Is this really needed?

Reply:
with this one changes that do not impact app wouldn't trigger building process. the thing with this is if nothing changed why build again